course_contents/2_intro_to_python/notes/03. Comprehensions.md

```diff
@@ -1,12 +1,13 @@
 ### The list comprehension
-- the list comprehension allows us to create a list without manually appending objects or running loops
+- the list comprehension allows us to create a list without explicitly appending objects or using loops
```

Comment:
I'm not sure about "not using loops", since there's a for loop inside a list comprehension 👀 

What about "Used to create a new list while iterating over a collection" ?

Reply:
I am happy to go with your suggestion instead of the line above, although I would prefer adding a part about not explicitly using a for loop to make it easier for students to remember what a comprehension is
If you don't think that the students will benefit from that then we can go with the exact sentence that you suggested